docs/GraFx-Studio/guides/template-variables/date/index.md

```diff
@@ -0,0 +1,110 @@
+# Date Variables
+
+## Create a Date Variable
+
+Under the Automate icon, click Variables, and add a variable with the "+" sign.
+
+![screenshot](date0.png)
+
+Choose "Date" as the Variable Type.
+
+## Set the Variable Name
+
+Double-click the name, or choose "Rename" under the "..." menu.
+
+![screenshot](date1.png)
+
+## Set General Properties
+
+![screenshot-full](date2.png)
+
+### Visible
+
+Defines if the variable will be visible in the [Studio UI](/GraFx-Studio/concepts/template-management/#studio-ui) to the end user.
+
+### Display Format
+
+A date can be displayed in several formats, depending on the local preferences. You can set a formatting string, how to display the date.
+
+The formatting can be empty, and will then default to "dd/MM/yyyy"
+
+**Format**
+
+Supported date formats
```

Comment:
They are actually date patterns, which can be used to specify a date format

Reply:
Check